backend/app/adapter/gqlapi/resolver/error.go

```diff
@@ -5,19 +5,21 @@ type ErrCode string
 
 // The constants enumerate all supported error codes.
 const (
-	ErrCodeUnknown            ErrCode = "unknown"
-	ErrCodeAliasAlreadyExist          = "aliasAlreadyExist"
-	ErrCodeRequesterNotHuman          = "requesterNotHuman"
-	ErrCodeInvalidLongLink            = "invalidLongLink"
-	ErrCodeInvalidCustomAlias         = "invalidCustomAlias"
-	ErrCodeAliasWithFragment          = "aliasWithFragment"
-	ErrCodeMaliciousContent           = "maliciousContent"
-	ErrCodeInvalidAuthToken           = "invalidAuthToken"
-	ErrCodeUnauthorizedAction         = "unauthorizedAction"
+	ErrCodeUnknown              ErrCode = "unknown"
```

Comment:
Let's format the code

Reply:
I believe I ran script formatter already, but I'll rerun it after I make my changes.